app/javascript/components/tree-view/reducers/index.js

```diff
@@ -3,11 +3,19 @@ import onCheck from './oncheck';
 import onClick from './onclick';
 import { checkAll, selectSilent } from './others';
 
+const actionsPrefix = (prefix) => {
+  const newObject = {};
+  Object.entries(ActionTypes).forEach(([key, obj]) => {
+    newObject[key] = prefix + obj;
+  });
+  return newObject;
+};
+
 export const ACTIONS = {
-  ...ActionTypes,
-  EMPTY_TREE: 'tree.empty',
-  CHECK_ALL: 'tree.checkAll',
-  SELECT_SILENT: 'node.select.silent',
+  ...actionsPrefix('@@tree/'),
+  EMPTY_TREE: '@@tree/tree.empty',
+  CHECK_ALL: '@@tree/tree.checkAll',
+  SELECT_SILENT: '@@tree/node.select.silent',
```

Comment:
Mixing dot-notation with camelCase is a little confusing. I guess we're using camelCase everywhere else, so we should stick to: `selectNodeSilently` or `selectNodeSilent` cc @himdel